recipes/nr.parsing.date/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "nr.parsing.date" %}
+{% set version = "1.0.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nr.parsing.date-{{ version }}.tar.gz
+  sha256: 473ae486a5bcdee77a8449dd9ab26d7b13a51f883c8c7106ae4b609dd26af385
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
+  run:
+    - nr.utils.re >=0.2.0,<1.0.0
```

Upstream lists and uses `dataclasses`. That can added as an unconditional dependency since the dataclasses package is empty on newer pythons:
```suggestion
    - nr.utils.re >=0.2.0,<1.0.0
    - dataclasses >=0.6.0,<1.0.0
```

---

Hey @dopplershift , what upstream are you talking about? With unconditional, you mean with version constraint?

---

Actually, it seems to be an issue with Grayskull (which I used to generate the recipes). It seems to omit any `dataclasses` requirements. All my packages in PyPI actually do list the dependency. I'll have to backfill it on a few more recipes.